core/src/main/scripts/importer/validateData.py

```diff
@@ -3835,6 +4087,14 @@ def index_geneset_id_list(json_data,
             result_list.append(geneset_id)
     return result_list
 
+def index_treatment_data(json_data,
+                         id_field = "treatmentId"):
```

Comment:
Styling: no spaces around keyword/default arguments unless combined with type annotations. ([PEP8](https://www.python.org/dev/peps/pep-0008/#other-recommendations))

Reply:
Also, I think strings in this file tend to use single quotes unless they're docstrings or likely to contain apostrophes.

Reply:
Changed to:
> def index_treatment_data(json_data,
>                         id_field='treatmentId'):

Also updated:
>def index_geneset_id_list(json_data,
>                         id_field='genesetId'):